pkg/network/ebpf/c/protocols/kafka/helpers.h

```diff
@@ -0,0 +1,287 @@
+#ifndef __KAFKA_HELPERS_H
+#define __KAFKA_HELPERS_H
+
+#include "protocols/helpers/big_endian.h"
+#include "protocols/kafka/defs.h"
+#include "protocols/kafka/maps.h"
+#include "protocols/kafka/types.h"
+
+// Wraps the mechanism of reading big-endian number (s16 or s32) from the packet, and increasing the offset.
+#define READ_BIG_ENDIAN_WRAPPER(type, name, skb, offset)    \
```

Comment:
Consider moving to the big endian file

Reply:
can't
the offset is unique to the logic in this file